patches_test.go

```diff
@@ -818,6 +731,14 @@ func TestPatchApply(t *testing.T) {
 	}
 }
 
+func MustObject(j string) map[string]any {
```

Comment:
Is this worth promoting to c/crossplane-runtime at some point in the future?

Reply:
I think so - I've definitely written it a few times now. I don't think it's super urgent though given how tiny it is.